native_client/java/README.rst

```diff
@@ -1,28 +1,42 @@
+.. _java-android-bindings:
 
 DeepSpeech Java / Android bindings
 ==================================
 
-This is still preliminary work. Please refer to ``native_client/README.rst`` for
-building ``libdeepspeech.so`` and ``deepspeech`` binary for Android on ARMv7 and
-ARM64 arch.
+Please refer to :ref:`Build documentation <build-native-client>` for building
+``libdeepspeech.so`` and ``deepspeech`` binary for Android on ARMv7 and ARM64
+arch.
```

Comment:
I think the "Using" section below should be the first thing on this file, then any build instructions.